pkgs/development/compilers/gcc/4.8/default.nix

```
@@ -167,6 +167,7 @@ assert x11Support -> (filter (x: x == null) ([ gtk2 libart_lgpl ] ++ xlibs)) ==
 
 stdenv.mkDerivation ({
   name = crossNameAddon + "${name}${if stripped then "" else "-debug"}-${version}";
```

should be able to do this as well then for all of the gcc derivations
```suggestion
  pname = crossNameAddon + "${name}${if stripped then "" else "-debug"}";
```

---

Oh, I forgot to mention this. The later gcc versions have the `crossNameAddon` at the _end_ of the name, after the version, so I wasn't sure what to make of it.

---

@oxij I see you moved the `crossNameAddon` to the start for the earlier versions. Are you aware of any reason I shouldn't change it for the later ones also?

cc @Synthetica9 @rardiol